src/widgets/BaseWindow.cpp

```diff
@@ -571,29 +584,6 @@ void BaseWindow::wheelEvent(QWheelEvent *event)
     }
 }
 
-void BaseWindow::onFocusLost()
```

Comment:
This is now handled directly in the `event` method using the renamed `windowDeactivateAction` member